adr/0010-web-dashboard-directory-structure-and-contexts.md

```diff
@@ -0,0 +1,79 @@
+# 10. Web dashboard directory structure and contexts
+
+Date: 2023-08-24
+
+## Status
+
+Proposal
+
+## Context
+
+The web dashboard repository, is a `phoenix` application, right now the directory structure and the overall `phoenix` framework convention regarding modules and code organization are not respected.
```

Comment:
`The web dashboard repository is a phoenix application. Right now, the directory`. A `.` looks better there.